clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/caching/view/model/KubernetesV2SecurityGroup.java

```diff
@@ -61,12 +61,16 @@
       ImmutableSet.of(EXTENSIONS_V1BETA1, NETWORKING_K8S_IO_V1BETA1, NETWORKING_K8S_IO_V1);
 
   private final KubernetesManifest manifest;
-  private final Keys.InfrastructureCacheKey key;
+  private final String account;
   private final String id;
 
   private final Set<Rule> inboundRules;
   private final Set<Rule> outboundRules;
 
+  public String getAccountName() {
```

Comment:
Any reason only security groups need this getter?

Reply:
All the others look things up by using the auto-generated `getAccount` accessor (and the deck interfaces only required `account` not `accountName`).  For some reason for security groups we've historically  [looked things up](https://github.com/spinnaker/clouddriver/blob/658bac1b8b21121cdb1ac6fbfcb76cec16d5a410/clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/SecurityGroupController.groovy#L54) by `accountName`, so it's become [required](https://github.com/spinnaker/clouddriver/blob/658bac1b8b21121cdb1ac6fbfcb76cec16d5a410/clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/SecurityGroup.java#L80) on the interface.

That being said, your question made me find a couple of places where `deck` reads `accountName` from a server group/server group manager even though it's not on the interface so I probably need to fix those before merging this ([here](https://github.com/spinnaker/deck/blob/cb71e811c49dc1e92f96045c436b4ce2bb1e167d/app/scripts/modules/kubernetes/src/serverGroup/details/details.html#L96) and [here](https://github.com/spinnaker/deck/blob/cb71e811c49dc1e92f96045c436b4ce2bb1e167d/app/scripts/modules/kubernetes/src/serverGroupManager/details/details.html#L102)).